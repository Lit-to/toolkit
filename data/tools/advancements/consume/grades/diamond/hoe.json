{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:diamond_hoe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/diamond/hoe"
    }
}