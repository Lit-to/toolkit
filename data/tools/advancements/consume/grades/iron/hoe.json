{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:iron_hoe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/iron/hoe"
    }
}