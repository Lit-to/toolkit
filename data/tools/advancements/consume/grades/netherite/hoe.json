{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:netherite_hoe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/netherite/hoe"
    }
}