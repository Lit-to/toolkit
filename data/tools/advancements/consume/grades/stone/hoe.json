{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:stone_hoe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/stone/hoe"
    }
}