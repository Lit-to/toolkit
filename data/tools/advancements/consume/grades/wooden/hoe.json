{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:wooden_hoe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/wooden/hoe"
    }
}