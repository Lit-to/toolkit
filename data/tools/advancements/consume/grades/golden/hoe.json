{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:golden_hoe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/golden/hoe"
    }
}